Drivers/Src/LTC681x.c

```diff
@@ -1370,6 +1368,11 @@ int16_t LTC681x_run_adc_redundancy_st(uint8_t adc_mode, uint8_t adc_reg, uint8_t
 
 	for (int cic=0; cic<total_ic; cic++)
 	{ 
+
```

Comment:
Why was this added?

Reply:
The ltc driver that we got from the LTC website did not implement the open wire checking function that well. The function would return a 64 bit where each bit represents an open wire on a certain pin but the cell_asic struct had a system_open_wire field per LTC6811 that was not being used even though it was defined in the struct. Originally it was confusing to just get a 64 bit number and not knowing which IC the open wire belonged to, especially since the LTC6811s aren't measuring 12 batteries. Having each IC have its own open_wire variable is easier to work with since we know which IC the open wire belonged to and it'll be scalable.